assets/src/edit-story/utils/useFocusOut.js

```diff
@@ -43,11 +43,11 @@ function useFocusOut(ref, callback, deps) {
     };
 
     node.addEventListener('focusout', onFocusOut);
-    node.ownerDocument.addEventListener('click', onDocumentClick);
+    node.ownerDocument.addEventListener('pointerdown', onDocumentClick);
```

Comment:
`useFocusOut` is currently used for:

* Text edit
* Dropdown
* Color picker
* Date picker

They all still work as intended with this change.